### PR TITLE
Inria fixes

### PR DESCRIPTION
**Pull request by ashnair1**
- Fixed `ignore_index` setting. 
- Added plot function

### CODE REVIEW

File: torchgeo/datamodules/inria.py [hunk 1]
```diff
@@ -243,3 +245,7 @@ def on_after_batch_transfer(
         if "mask" in batch:
             batch["mask"] = rearrange(batch["mask"], "b () h w -> b h w")
         return batch
+
+    def plot(self, *args: Any, **kwargs: Any) -> plt.Figure:
+        """Run :meth:`torchgeo.datasets.InriaAerialImageLabelingDataModule.plot`."""
+        return self.dataset.plot(*args, **kwargs)
```
Comment (ashnair1) [inline torchgeo/datamodules/inria.py:251]: @adamjstewart Simplified the plot function as suggested in the object detection PR.

File: conf/inria.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ experiment:
     learning_rate_schedule_patience: 6
     in_channels: 3
     num_classes: 2
-    ignore_index: 0  # class 0 not used for scoring
+    ignore_index: null
```
Comment (adamjstewart) [inline conf/inria.yaml:24]: @calebrob6 can you double check this? It relates to #644/#444.

File: torchgeo/datamodules/inria.py [hunk 3]
```diff
@@ -243,3 +245,7 @@ def on_after_batch_transfer(
         if "mask" in batch:
             batch["mask"] = rearrange(batch["mask"], "b () h w -> b h w")
         return batch
+
+    def plot(self, *args: Any, **kwargs: Any) -> plt.Figure:
+        """Run :meth:`torchgeo.datasets.InriaAerialImageLabeling.plot`."""
```
Comment (adamjstewart) [inline torchgeo/datamodules/inria.py:250]: This should have a `.. versionadded:: 0.4`

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]

File: conf/inria.yaml (hunk 2 above)
Comment (calebrob6) [reply conf/inria.yaml:24]: looks good to me

**Pull request merged**